ONBOARDING.md

```diff
@@ -0,0 +1,84 @@
+- [Introduction](#introduction)
+  - [Who this is for](#who-this-is-for)
+  - [What this guide will help you do](#what-this-guide-will-help-you-do)
+- [Learn key concepts](#learn-key-concepts)
+  - [About the OpenSearch project](#about-the-opensearch-project)
+  - [How we work](#how-we-work)
+  - [Contributor expectations](#contributor-expectations)
+  - [Resources and links](#resources-and-links)
+- [Set up accounts and access](#set-up-accounts-and-access)
+  - [GitHub account](#github-account)
+  - [OpenSearch forum account](#opensearch-forum-account)
+
+# Getting started as an OpenSearch contributor
+
+## Introduction
+
+### Who this is for
+
+Any developer who’s interested in contributing code (features and fixes) to one of the OpenSearch open source project repositories for the first time.
+
+### What this guide will help you do
+
+1. Understand the key concepts of the OpenSearch project and how to find additional resources
+2. Set up the accounts you’ll need to contribute
+
+## Learn key concepts
+
+### About the OpenSearch project
+
+- **Read (3 min) [the project homepage](https://opensearch.org/)** to understand to get a sense of what the OpenSearch software does and how it’s used
+- **Read (4 min) “[About OpenSearch](https://opensearch.org/about.html)”** to understand the primary project components and development principles
+- Optional - Read founding documents to understand the project history
+  - Read (8 min) “[Introducing OpenSearch](https://aws.amazon.com/blogs/opensource/introducing-opensearch/)” from 2021-04-12
+  - Read (6 min) “[Stepping up for a truly open source Elasticsearch](https://aws.amazon.com/blogs/opensource/stepping-up-for-a-truly-open-source-elasticsearch/)” from 2021-01-21
+
+### How we work
+
+We model our way of working on “[the Apache Way](https://apache.org/theapacheway)” (optional - read 4 min). Specifically, we’ve borrowed and adapted:
```

Comment:
I would have preferred to just link to this as a reading, but it's pretty apache-specific and didn't capture the ways we've adapted for our project. But I could imagine this info living elsewhere. It's worth noting that this section is not original work, but I believe I've adapted and referenced it appropriately.